### ISSUE TITLE
Web Accessibility issues

### ISSUE THREAD

**Issue by briri** [substance/forms#3]
The following are some basic web accessibility issues our UX team discovered with substance-forms. 
- The toolbar does not have enough color contrast to support. Converting to a simple black and white color scheme would solve this issue
- Screen readers describe each of the toolbar icons as 'button'
- When text is highlighted there is no prompt to the screen reader that lets the user know that a toolbar appeared
- When the user clicks on the comment icon there is no audible indication that a comment area appeared

**Comment by michael** [comment]
- I agree on the color contrast. Need to improve that, I wanted to avoid introducing additional colors and use grey for inactive, white for the active state. If we go with white color then we need a different indicator for active buttons. May just come at the cost of being less visually appealing.

![image](https://cloud.githubusercontent.com/assets/2931/24906627/c3d8f8d2-1eb8-11e7-80d0-dacffb0d83c7.png)

- As for screen readers, we haven't addressed them yet. Think it would be good to design that with accessibility experts and do user testing. I think we would need a bit of time to do that properly. Could you ask your UX team to give advice on how we should change to markup to improve accessibility?

**Comment by stephaniesimms** [comment]
Our UX team isn't available for a full accessibility evaluation until after our MVP (late summer). At that point we can do the work and contribute it back to you.
